src/ircmessageformatter.cpp

```diff
@@ -0,0 +1,447 @@
+/***************************************************************************
+ *   Copyright (C) 2008-2017 The Communi Project                           *
+ *   Copyright (C) 2017 by Fae - itsthefae@gmail.com                       *
```

Comment:
This file was taken in part from the examples within the library, and they have their own license notice that states they are not covered by the BSD-3-Clause license as in the rest of the library.   
See this page with the code and notice:  http://communi.github.io/doc/3.5/ircmessageformatter_8cpp.html

The sources maintain the original copyright, since they are stored in their own 3rd-party file and are not modified by our project.  However, if the project doesn't already include the notice in its about section and a copy of the license file for Communi library isn't already made accessible with the binary releases, it probably should be added.